nautobot/ipam/models.py

```diff
@@ -1066,6 +1065,25 @@ def _deconstruct_address(self, address):
 
     natural_key_field_names = ["parent__namespace", "host"]
 
+    def _get_closest_parent(self):
+        if self.present_in_database:
+            namespace = self._namespace or self.parent.namespace
+        elif self._namespace is None:
+            raise ValidationError({"parent": "Namespace could not be determined."})
```

Comment:
When would this error be triggered? I don't see it in the tests.

Reply:
if neither `namespace` nor `parent` is specified when an IPAddress is created e.g `IPAddress.objects.create(address="...", status="...")`